lib/export.js

```diff
@@ -61,20 +63,26 @@ class FHIRExporter {
   }
 
   export() {
-    this._codeSystemExporter.export();
+    logger.info(`Exporting FHIR using target: ${this._target}`);
+    if (!(this._target === 'FHIR_DSTU_2')) {
+      this._codeSystemExporter.export();
+    }
     this._valueSetExporter.export();
```

Comment:
Just for my sanity, to make sure I understand: We don't need to have a separate functionality for the value set exporter, because all version-dependent changes relevant to a value set would be within the code systems it contains. Is that correct?

Reply:
I'm not sure I understand the question.  Here's the deal.  In STU3 (and above), you define new code systems using a `CodeSystem` resource.  In DSTU2, there is no `CodeSystem` resource -- instead, you define new code systems _inline_ in a `ValueSet`.  For this reason, we don't use the `CodeSystemExporter` for DSTU2.  Instead, the `ValueSetExporter` will detect it is DSTU2 and put the code system inline when necessary.

Reply:
Thanks for the clarification. Given this explanation, and my realization in a comment below on the value set exporter, this all makes sense to me.